appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -1529,12 +1529,26 @@
 
   // Used in explorer/commands/ShowBarcodeCommand.java
 
-  @DefaultMessage("Barcode link for {0}")
-  @Description("Title of barcode dialog.")
-  String barcodeTitle(String projectName);
+  @DefaultMessage("Android App for {0}")
+  @Description("Title of download apk dialog.")
+  String downloadApkDialogTitle(String projectName);
 
-  @DefaultMessage("Note: this barcode is only valid for 2 hours. See {0} the FAQ {1} for info " +
-      "on how to share your app with others.")
+  @DefaultMessage("Android App Bundle for {0} (to be used in Google Play Store)")
+  @Description("Title of download aab dialog.")
+  String downloadAabDialogTitle(String projectName);
+
+  @DefaultMessage("Download .apk now")
+  @Description("Download button shown in barcode dialog")
+  String barcodeDownloadApk();
+
+  @DefaultMessage("Download .aab now")
+  @Description("Download button shown in barcode dialog")
+  String barcodeDownloadAab();
+
+  @DefaultMessage("<b>Click the button to download the app, right-click on it to copy a download link, or scan the " +
```

Comment:
Humm, it will probably show the older version in the other languages until someone manually changes it. I have now moved it into a `barcodeWarning2` key, to always force to display the updated one (although it will require a translation anyway) and don't show different content depending on language.